README.md

```diff
@@ -1,3 +1,5 @@
+![FontBoy screenshot](FontBoy.png "FontBoy")
```

Comment:
It'd look better after the introduction.
While you're at it, remove the "Version 0.9.7". Future release may not need a changed readme.

Reply:
I updated it